CSIRO/CARS/incoming_handler.sh

```diff
@@ -15,7 +15,7 @@ regex_filter() {
 # $1 - file to handle
 main() {
     local file=$1; shift
-    local regex="^CARS[[:digit:]]{4}_(World_monthly|Australia_weekly)\\.nc$"
+    local regex="^CARS[[:digit:]]{4}_(World_monthly|Australia_weekly).*\\.nc$"
```

Comment:
Ok, then how about:
1. We delete and unharvest the original monthly file,
2. we change the regex everywhere needed,
3. we upload both original and splitted monthly files and the splitted files only should be harvested.